src/yunohost/backup.py

```diff
@@ -2231,9 +2235,16 @@ def backup_info(name, with_details=False, human_readable=False):
     if not os.path.exists(info_file):
         tar = tarfile.open(archive_file, "r:gz")
         info_dir = info_file + '.d'
-        tar.extract('info.json', path=info_dir)
-        tar.close()
-        shutil.move(os.path.join(info_dir, 'info.json'), info_file)
+        try:
+            tar.extract('info.json', path=info_dir)
+        except KeyError:
+            logger.debug("unable to retrieve '%s' inside the archive",
```

Comment:
I think we should display this as warning such that user can see it. Otherwise he/she'll have to rerun `--debug` to understand what's going on. (But maybe it's okay like this, idk)